Packs/OpenLDAP/Integrations/OpenLDAP/OpenLDAP.yml

```diff
@@ -85,6 +85,19 @@ configuration:
   - Start TLS
   required: true
   type: 15
+- additionalinfo: The SSL\TLS version to use in SSL or Start TLS connections. Default is None.
```

Comment:
Default is None doesnt really explain what will happen when I dont set this param. None should be part of the dropdown and default. We should give it a better name than None

Reply:
Thank you.
I improved the description. Added a recommendation to use the TLS_CLIENT version with a link to a reference. 
Please Let me know what you think.